editor/plugins/org.fusesource.ide.wsdl2rest/pom.xml

```diff
@@ -105,6 +108,17 @@
 			<groupId>log4j</groupId>
 			<artifactId>log4j</artifactId>
 		</dependency>
+        <dependency>
```

Comment:
indentation issue

Reply:
The indention is not causing the build failures. :)

Reply:
yes but it needs to be fixed before merging